docs/references/architecture/adr-114-undo-109.md

```diff
@@ -92,11 +92,11 @@ Column legend:
 | bits                       | keep private       | 0                                      | 26             | low        | [bits-url]        |
 | blocksync                  | keep private       | 0                                      | 6              | low        | [blocksync-url]   |
 | clist                      | keep private       | 0                                      | 24             | low        | [clist-url]       |
+| indexer                    | keep private       | 0                                      | 0              | low        | [indexer-url]     |
```

Comment:
The indexer module has 0 public uses, so it doesn't need to become public again. We can keep it internal.